client/browser/README.md

```diff
@@ -18,8 +18,8 @@ Supported code hosts:
 
 After installing it in Chrome, visit the following pages to see it in action:
 
-- [Example code file on GitHub](https://github.com/sourcegraph/sourcegraph/blob/main/cmd/repo-updater/internal/repoupdater/observability.go)
-- [Example pull request on GitHub](https://github.com/sourcegraph/sourcegraph/pull/58878/files)
+- [Example code file on GitHub](https://github.com/sourcegraph/sourcegraph-public-snapshot/blob/main/internal/repoupdater/client.go)
+- [Example pull request on GitHub](https://github.com/sourcegraph/sourcegraph-public-snapshot/pull/58878/files)
```

Comment:
@sqs `cmd/repo-updater/internal/repoupdater/observability.go` was deleted. I poked around a bit and chose `internal/repoupdater/client.go` somewhat arbitrarily.